flow/project.py

```diff
@@ -418,11 +435,20 @@ def __init__(self, cmd, pre=None, post=None, directives=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def eligible(self, job):
-        "Eligible, when all pre-conditions are true and at least one post-condition is false."
-        pre = all(cond(job) for cond in self._prereqs)
+    def eligible(self, job, ignore_conditions='none'):
+        """Eligible, when all pre-conditions are true and at least one post-condition is false,
+        or corresponding conditions are ignored """
```

Comment:
Please follow conventions about doc-string formatting outlined in the active [PEP 257](https://www.python.org/dev/peps/pep-0257).

@mikemhenry I think it will be crucial streamline our development workflow to automatically lint for these formatting issues as soon as possible.